server/src/service/machinesSchema.ts

```diff
@@ -0,0 +1,75 @@
+import { gql } from "apollo-server-express";
+
+export const MachinesTypeDefs = gql`
+  type Machine {
+    id: ID!
+    machineFamily: Int!
+    name: String!
+    room: String!
+    addedAt: Date
+    inUse: Boolean!
+  }
+
+  type MachineFamily {
+    id: ID!
+    name: String!
+    description: String
+    trainingModule: Int
```

Comment:
instead of having the `trainingModule` be an `Int` you have it be the actual `trainingModule` type

Reply:
@pjw9963 in that case would the input have to be a full `trainingModule` object instead of just an identifier?

Reply:
inputs cant have full types, only scalar values. so I would have the input have a trainingModule identifier to an existing training module